api/python/arrus/devices/us4r.py

```diff
@@ -282,6 +282,10 @@ def get_afe(self, addr):
         """
         return self._handle.getAfe(addr)
 
+    def hvps_set_voltage(self, voltage):
```

Comment:
Why do we need one more function to set voltage? (`set_hv_voltage` is not enough?)

Reply:
Used for debug purposes, now removed